### DISCUSSION TITLE
ReactiveRecord and ReactiveObject

### DISCUSSION THREAD

**Discussion by marcoablanco** [reactiveui/ReactiveUI#3445] [Q&A] [answered]
Why exist both? They have the same summary and same code?

https://github.com/reactiveui/ReactiveUI/blob/main/src/ReactiveUI/ReactiveObject/ReactiveObject.cs#L14
https://github.com/reactiveui/ReactiveUI/blob/main/src/ReactiveUI/ReactiveObject/ReactiveRecord.cs#L14
```xaml
/// <summary>
/// ReactiveObject is the base object for ViewModel classes, and it
/// implements INotifyPropertyChanged. In addition, ReactiveObject provides
/// Changing and Changed Observables to monitor object changes.
/// </summary>
```

ReactiveObject will be deprecate?

**Comment by glennawatson** [comment, marked as answer]
Ones for classes. Ones for records. Records can't inherit from classes. C# 9 feature.
